### PR TITLE
remove extraneous local variable usage in newETCD3Client

### PR DESCRIPTION
**Pull request by gorilla001**
**What this PR does / why we need it**:
code style minor fix

**Which issue(s) this PR fixes** 
none

**Special notes for your reviewer**:
none
**Release note**:

```release-note

```

### CODE REVIEW

**Comment by neolit123** [comment]
@gorilla001 
thanks for the PR.
LGTM.

/release-note-none
/ok-to-test
/kind cleanup

File: staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3.go [hunk 1]
```diff
@@ -104,8 +104,8 @@ func newETCD3Client(c storagebackend.Config) (*clientv3.Client, error) {
 		Endpoints: c.ServerList,
 		TLS:       tlsConfig,
 	}
-	client, err := clientv3.New(cfg)
-	return client, err
```
Comment (yue9944882) [inline staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3.go]: > minor fix

hi @gorilla001 thx for contributing. whats the fix btw? didn't really understand.

Comment (krmayankk) [reply staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3.go]: there is no fix, its a minor style correction, but nonetheless a good one.

Comment (gorilla001) [reply staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3.go]: @yue9944882 yes, its only a style correction, not a fix. @krmayankk

**Comment by jpbetz** [comment]
/lgtm

I've renamed the issue for posterity.

**Comment by nikhita** [comment]
/assign @liggitt 
for approval

**Pull request merged**